config/v1/types_cluster_version.go

```diff
@@ -95,6 +95,13 @@ type ClusterVersionStatus struct {
 	// +required
 	Desired Release `json:"desired"`
 
+	// architecture indicates whether the cluster is a homogeneous or
```

Comment:
If it's an enum, it should have enumerated values.

Reply:
As noted in comment below, the payload will have new metadata set to `multi` only for heterogeneous payloads. All other payloads are unchanged. For the homogeneous payloads CVO will determine arch using runtime.GOARCH. So I would think that the enum values would be `multi` + the valid OpenShift architectures.